extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -521,17 +519,7 @@ export class Visibility {
         ? Date.now() - perf.timing.domInteractive
         : '';
 
-    // Calculate the amount element visible at the time page was loaded. To do
-    // this, assume that the page is scrolled all the way to top.
-    const viewportRect = {top: 0, height: rb.height, left: 0, width: rb.width,
-        bottom: rb.height, right: rb.width};
-    const elementRect = {top: ir.top, left: ir.left, width: br.width,
-      height: br.height, bottom: br.height, right: br.width};
-    const intersection = rectIntersection(viewportRect, elementRect);
-    state[LOAD_TIME_VISIBILITY] = intersection != null
-        ? Math.round(intersection.width * intersection.height * 10000
-              / (br.width * br.height)) / 100
-        : 0;
+    state[LOAD_TIME_VISIBILITY] = state[LOAD_TIME_VISIBILITY] || 0;
```

Comment:
When we set it above, is there any harm in setting it to false if it's `>= 300` since load time? Ie:

```js
if (visible > 0) {
  // ...

  // Consider it as load time visibility if this happens within 300ms of
  // page load.
  if (state[LOAD_TIME_VISIBILITY] == undefined) {
    state[LOAD_TIME_VISIBILITY] =  timeElapsed < 300 ? visible : 0;
  }
}